eng/testing/tests.mobile.targets

```diff
@@ -146,38 +146,37 @@
 
   </Target>
 
-  <UsingTask TaskName="WasmAppBuilder"
-             AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
+  <Import Project="$(MonoProjectRoot)\wasm\build\WasmApp.targets" Condition="'$(TargetOS)' == 'Browser'" />
```

Comment:
Could this be moved to the bottom (convention of putting target imports at the end of the file)?

Reply:
I can get this in a follow up PR.